Extensions/TeamCity/Src/vss-extension.json

```diff
@@ -201,7 +211,8 @@
           },
           {
             "target": "artifactDetails",
-            "resultTemplate": "{ Name: \"{{version}}\", StreamType: \"zip\", downloadUrl : \"{{endpoint.url}}downloadArtifacts.html?buildTypeId={{definition}}&buildId={{version}}\" }"
```

Comment:
It gets used as a part of download https://github.com/Microsoft/vsts-agent/blob/master/src/Agent.Worker/Release/Artifacts/CustomArtifact.cs#L91